build/build.ps1

```diff
@@ -7,11 +7,37 @@ $ErrorActionPreference = 'Stop'
 $all_ok = $True
 Write-Host "Assembly version: $Env:ASSEMBLY_VERSION"
 
-choco install llvm --version=11.1.0 --allow-downgrade
-if (!(Get-Command clang -ErrorAction SilentlyContinue) -and (choco find --idonly -l llvm) -contains "llvm") {
-    # For some reason, adding to the path does not work on our build servers, even after calling refreshenv.
-    # LLVM was installed by Chocolatey, so add the install location to the path.
-    $env:PATH += ";$($env:SystemDrive)\Program Files\LLVM\bin"
+if (($IsWindows) -or ((Test-Path Env:/AGENT_OS) -and ($Env:AGENT_OS.StartsWith("Win")))) {
+    if (!(Get-Command clang        -ErrorAction SilentlyContinue) -or `
```

Comment:
Why do we need clang and clang-format? Is it just for the development project's sample?

Reply:
I'm going to try an approach that removes the need for installing these. Here's hoping it works...

Reply:
I think I have a viable approach here, let me know what you think. I'll double check with @bettinaheim to make sure this makes sense too.